### ISSUE TITLE
Highlighting broken using helper function in v1.4.0

### ISSUE THREAD

**Issue by adbutterfield** [styled-components/vscode-styled-components#256]
**Describe the bug (including copyable syntax)**
We use helper functions for breakpoints like so:
```
export const Layout = styled.div<{ bgColor: string }>`
  width: 100%;
  background-color: ${(props) => props.bgColor};
  ${breakpoints.mediumDown(css`
    padding: 0 0.5em;
  `)}
`;

export const LayoutInner = styled.div`
  max-width: 1024px;
  margin: 0 auto;
  ${breakpoints.mediumDown(css`
    margin: 0 1em;
  `)}
`;
```

With v1.4.0 the highlighting is broken, affecting the first component, and all subsequent components. See "Not working" screenshot.

If I change the code to put the )} on a new line, the syntax highlighting does work correctly (but my eslint rules don't like it). See "Working but eslint complains" screenshot. Code for "Working but eslint complains":
```
export const Layout = styled.div<{ bgColor: string }>`
  width: 100%;
  background-color: ${(props) => props.bgColor};
  ${breakpoints.mediumDown(css`
      padding: 0 0.5em;
    `
  )}
`;

export const LayoutInner = styled.div`
  max-width: 1024px;
  margin: 0 auto;
  ${breakpoints.mediumDown(css`
      margin: 0 1em;
    `
  )}
`;
```

**Screenshot**
Not working:
<img width="528" alt="Screen Shot 2020-12-08 at 11 27 54" src="https://user-images.githubusercontent.com/4752047/101430588-a1358580-3948-11eb-9261-1822fd6097e7.png">

Working but eslint complains:
<img width="538" alt="Screen Shot 2020-12-08 at 11 47 27" src="https://user-images.githubusercontent.com/4752047/101431899-2e79d980-394b-11eb-91a5-94a18d40a72e.png">

**To Reproduce**
First code example will reproduce.

**Expected behavior**
Correct syntax highlighting should be applied, no matter if there is a new line before the )}.

**Build environment (please complete the following information):**

- OS: macOS Catalina 10.15.7
- VSCode Version: 1.51.1
- Extension Version 1.4.0

**Additional context**

**Comment by joakim-roos** [comment]
Exactly the same issue here!

**Comment by MicaelRodrigues** [comment]
The same here (In this case breaks using `ifProp` from [styled-tools](https://github.com/diegohaz/styled-tools):

<img width="750" alt="Screen Shot 2020-12-10 at 14 36 58" src="https://user-images.githubusercontent.com/796443/101785855-29f12480-3af5-11eb-822b-89561241fb2c.png">

**Comment by jasonwilliams** [comment]
Fixed in v1.5.0
